modules/pwease.py

```diff
@@ -0,0 +1,18 @@
+from .base import Module
+	
+class PWEASE(Module):
+	def responseMaker(message):

```

Comment:
I don't see a reason to have this as a separate function. Also, use underscore_case in Python.